frontend/pages/club/[club]/renew.tsx

```diff
@@ -222,7 +222,9 @@ const RenewPage = (props: RenewPageProps): ReactElement => {
   }
 
   if (SITE_ID === 'fyh') {
-    return <ResourceCreationPage {...props} />
+    return (
+      <ResourceCreationPage {...props} student_types={props.studentTypes} />
```

Comment:
It would be better to modify `ResourceCreationPage` to accept `studentTypes` instead of `student_types` than doing this.

Reply:
So `ResourceCreationPage` now accepts `studentTypes` in camel case, but seems like `create.tsx`'s `doBulkLookup` method was feeding it with `student_types`. So I tried using `[string, string][]` form argument to rename the request data, but seems like it was broken(`doBulkLookup` won't work with `[string, string][]`; specifically, it would fail to parse the json). This branch now contains the fix, so we should probably check if there are any other parts that depend on this functionality.